src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -1100,7 +1101,7 @@ CLASS ZCL_ABAPGIT_OBJECTS IMPLEMENTATION.
       APPEND <ls_result> TO rt_results.
     ENDLOOP.
 
-* TOBJ has to be handled before ODSO
+* TOBJ has to be handled before SCP1
```

Comment:
Does this line belong to this PR?

Reply:
Just fixing the comment for https://github.com/larshp/abapGit/pull/3543 (typical cut&paste mistake)